build_latest.sh

```diff
@@ -1,5 +1,12 @@
 #!/bin/sh
+# Builds the latest Indico image
+
+if ! command -v jq >/dev/null 2>&1
+then
+    echo 'jq is not installed'
+    exit
+fi
 
 LATEST_VERSION=$(curl --insecure https://api.github.com/repos/indico/indico/releases/latest | jq '.name[1:]' --raw-output)
-echo Building Indico ${LATEST_VERSION}...
+echo 'Building Indico ${LATEST_VERSION}...'
```

Comment:
Are you sure this works? Usually variables are not substituted inside single-quoted strings.

Reply:
yep good catch